scripts/redcap/wrong_date_associations.py

```diff
@@ -148,7 +150,41 @@ def log_row(row: pd.Series, **kwargs):
     for _, row in errors_df.reset_index().iterrows():
         log_row(row, uid=uid_template, **kwargs)
 
+def get_special_cases(marks, session):
+    """
+    Looks at special_cases.yml file, finds the special cases under the file name section,
+    and then excludes any cases that match the special cases list.
+
+    Note (for Chris use):
+    - 'redcap_event_name' is arm name
+    - 'study_id' inclues up to letter and number
+    - 'visit_date' is looking at date --> not correct!
```

Comment:
I have no idea what this means?

Reply:
More of a personal note, for me. In terms of the `not correct!` part, this was mainly due to realizing that I wanted to look at a date field for comparison and not an actual date (i.e. the `bionp_date` as opposed to `2021-07-19`)